tests/steps/gdc_pdc_TCGA-E2-A10A_merged.yaml

```diff
@@ -21,95 +21,165 @@ id: "TCGA-E2-A10A"
 ethnicity: "Not Hispanic or Latino"
 sex: "Female"
 race: "White"
+taxon: "Homo sapiens"
+days_to_birth: "-15085"
 
 # Research subjects were simply appended
 ResearchSubject:
   - id: "4da7abaf-ac7a-41c0-8033-5780a398545c"
-    identifier: "TCGA-E2-A10A"
+  # identifier should be an array.  And CDA_context would go away.
+  # In json schema:
+  #  "identifier": [
```

Comment:
@kreinold , can you further clarify the goal with having this structure?

Reply:
Aligns with CCDH data model

Reply:
> In all cases it is intended to point to the same entity in other contexts
I was having the same understanding, but I'm confused as to how context-specific case_id (line 35) represents a link to external world?

Reply:
Another thing that I really can't wrap my head around is the concept of Case/ResearchSubject that is shared among systems. I can totally understand how Patient and Specimen can be a common entities, but I thought that the Case is simply a logistical piece of information -- it's a descriptor of context in which a particular Patient/Specimen was analyzed. Am I missing something here?

Reply:
3 points -- the first 2 being the most relevant; the third is a future and not in scope.  
1. The property 'identifier' is used throughout the CCDH model with the same definition and format; it's a data type.  We shouldn't use it differently in one or two contexts.  
2.  Adding our own CDA_context rather than using the CCDH model for capturing this information is an unnecessary deviation from CCDH.  (We're essentially just renaming a field.)
3. This one's a long story -- and not a concern at this time.  Much of the data in CDA -- TCGA is a good example -- has been used elsewhere and analysis results derived from this data appears in other data repositories.  It is possible that the data was analysis based on sequencing done within the context of a project; the only link may be between the case_id and the external repository.